FriendsWave/social/models.py

```diff
@@ -1,3 +1,90 @@
+from functools import _Descriptor
 from django.db import models
+from django.db.models.base import Model
+from django.db.models.fields.related import OneToOneField
+
+class Profil(models.Models):
+    pseudo = models.CharField(max_length=100)
+    image = models.ImageField()
+    user = models.ForeignKey('auth.User', on_delete=models.CASCADE)
+    friends = models.ManyToManyField('self', through='Friend')
+    topics = models.ManyToManyField('Topic', through='UserTopic')
+    etat = models.BooleanField(default=False)
```

Comment:
etat field is meaning less. active, is_active, profile_is_active are increasingly meaningful